api-reference/v1.0/api/directory_deleteditems_list.md

```diff
@@ -0,0 +1,97 @@
+# List deleted items
+
+Retrieve a list of recently deleted items from [deleted items](../resources/directory.md).
+
+Currently, deleted items functionality is only supported for the [group](../resources/group.md) and [user](../resources/user.md) resources.
+
+## Permissions
+One of the following permissions is required to call this API. To learn more, including how to choose permissions, see [Permissions](../../../concepts/permissions_reference.md).
+
+### For users:
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All, Directory.AccessAsUser.All |
+|Delegated (personal Microsoft account) | Not supported. |
+|Application | User.Read.All, User.ReadWrite.All, Directory.Read.All, Directory.ReadWrite.All |
+
+### For groups:
+
+|Permission type      | Permissions (from least to most privileged)              |
+|:--------------------|:---------------------------------------------------------|
+|Delegated (work or school account) | Group.Read.All, Group.ReadWrite.All, Directory.Read.All, Directory.AccessAsUser.All |
+|Delegated (personal Microsoft account) | Not supported.    |
+|Application | Group.Read.All, Group.ReadWrite.All, Directory.Read.All |
+
+## HTTP request
+<!-- { "blockType": "ignored" } -->
+```http 
+GET /directory/deleteditems/Microsoft.Graph.Group
```

Comment:
While this work with initial uppercase for the types, you should make these all lowercase
`GET /directory/deleteditems/microsoft.graph.group`

This should be done for all topics too.  Please fix this in the Beta docs (but you can do this as a separate bug fix PR)